README.md

```diff
@@ -41,28 +41,27 @@ The figure below illustrates the software flow through the architecture componen
 
 # Getting Started
 
-To set up your system, follow [this guide.](./docs/setup.md)  
-For a full walkthrough on how to use this service and RTSP streaming, follow [this guide.](./docs/general-usage.md)  
+Learn how to configure and run the service by following these [instructions](./doc/setup.md). 
```

Comment:
I would check these links. They point to the `doc` folder, but the folder in usb is `docs, and so many of them are failing. However, this points out another issue, that ONVIF uses `doc` as the documentation directory and USB uses `docs`. Should we standardize this between services?